specs/rollup-node.md

```diff
@@ -175,58 +235,77 @@ The following scenarios are assimilated to errors:
 - [`engine_executePayloadV1`] returning a `status` of `"SYNCING"` or `"INVALID"` whenever passed an execution payload
   that was obtained by a previous call to [`engine_getPayloadV1`].
 
-## Handling L1 Re-Orgs
+### Finalization Guarantees
 
```

Comment:
I added mentions to fault proofs here because this section makes little sense without it. I'm also okay to remove this this section altogether to preserve consistency (though doing so make the prescriptions on the `forkchoiceState` seemingly arbitrary).